### ISSUE TITLE
Menu broken to Armbian System Settings

### ISSUE THREAD

**Issue by ChiefGyk** [monero-ecosystem/PiNode-XMR#11]
It seems to be pointing to Raspbian and calling it that despite me having chosen the Armbian setup for my Odroid HC2

**Comment by shermand100** [comment]
Just double checking you're referring to the Menu that appears in the terminal when you log in?

I've corrected the menu for Armbian-config (not raspbian-config) just now.

Lots of new features were added when I did some development on the Raspberry Pi branch and nearly all of them are functional on the  Armbian branch as Raspberry Pi OS Buster and Armbian Debian Buster are so similar.

I will go through those menus though and clean up their sources so future development is separated between branches and will comment and close here when complete.
